test/index.test.js

```diff
@@ -132,6 +132,49 @@ describe('guardian.js', function () {
         });
       });
 
+      describe('when no transaction is returned on reply', function () {
+        let response;
+
+        beforeEach(function () {
+          response = {
+            deviceAccount: {
+              methods: ['otp'],
+              availableMethods: ['otp'],
+              name: 'test',
+              phoneNumber: '+1234'
+            },
+            availableEnrollmentMethods: ['otp'],
+            availableAuthenticationMethods: ['push']
+          };
+
+          socketClient.connect.yields();
+          httpClient.post.yields(null, response);
+
+          guardianjs = guardianjsb({
+            serviceUrl: 'https://tenant.guardian.auth0.com',
+            requestToken,
+            issuer: {
+              label: 'label',
+              name: 'name'
+            },
+            accountLabel: 'accountLabel',
+            globalTrackingId: 'globalTrackingId',
+            dependencies: {
+              httpClient,
+              socketClient
+            }
+          });
+        });
+
+        it('start fails with invalid token error', function (done) {
```

Comment:
Minor: should read well including the "it" as first word "it start fails with invalid token error" does not read so well to me, what about "callbacks with invalid token error" and I think you are already in the context of start so you could leave it like that or add a `describe('when transaction token decode fails with invalid token error`, () => ...)`

Reply:
Agreed, changed.